src/main/java/teammates/common/util/Const.java

```diff
@@ -983,6 +984,7 @@ private Const() {
         public static final String RESPONSES = "/responses";
         public static final String HAS_RESPONSES = "/hasResponses";
         public static final String SUBMISSION_CONFIRMATION = "/submission/confirmation";
+        public static final String REGENERATE_STUDENT_COURSE_LINKS = "/student/uniquelinks/regeneration";
```

Comment:
Make the URL consistent with the string.

`Student unique link` or `student course link`?

Reply:
Since it is **`/student`**`/...`, I think `courselinks/regeneration` will be suitable. It only regenerates the student's (unique) links for a particular course, not all of the student's unique links.